src/strategyHelpers/blueGreen/servicesBlueGreenHelper.test.ts

```diff
@@ -0,0 +1,7 @@
+// deployManifests
```

Comment:
remove this file. Probably an intermediate stage with the larger refactor but need to keep things clean